06-building-a-program-start-to-finish/writing-tests/output/inverted-index.spec.js

```diff
@@ -0,0 +1,92 @@
+
```

Comment:
Nice one fixing the errors. However, your suites and specs do not follow BDD guidelines.

Your suites when combined with your specs should make a proper sentence.

You currently have: `Test suite for helper functions`...`Given an unordered object, function should return an alphabetically ordered version of the object`

I'd probably refactor this as:

`Sort function`...`should return an alphabetically ordered version of the object`

PS: With this you'll not have all these lines exceeding 80 characters.

Please re-write, I also don't see you testing for the exact index that was created and also for the exact search results returned for one file and then when you search through all files.